content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Introducing the Istio Operator
+description: Introduction to Istio's new operator-based installation and control plane management feature.
+publishdate: 2019-11-04
+subtitle:
+attribution: Martin Ostrowski (Google), Frank Budinsky (IBM)
+keywords: [install,configuration,istioctl,operator]
+target_release: 1.4
+---
+
+Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
+a pattern for encoding human operations knowledge in software and are a popular way to simplify
+the administration of software infrastructure components.
+Because Istio is complex to administer, it's a natural candidate for an operator.
+
+Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
+This approach worked well but had some disadvantages:
+
+1. Users need to install another tool.
+1. Supporting Istio's many small specializations cause Helm templates to become unwieldy.
+1. Helm configurations are difficult to validate using Istio's proto-based programming model.
+
+With operators, the API is schematized in a
+[Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
```

Comment:
schematized in a [CustomResourceDefinition]() which is generated from a [proto](/docs/reference/config/istio.operator.v1alpha12.pb/)